index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
Great job @Yidnekachew-cmd 🤝

- [ ] [Optional] As stated in the project requirements, you could have added a class to the HTML element and then apply styles to this element through this class in your styles file.